tests/phpunit/CRM/Mailing/TokensTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @group headless
+ */
+class CRM_Mailing_TokensTest extends \CiviUnitTestCase {
```

Comment:
Mostly because I reflexively want to reason about tests at the suite level -- e.g. "all the tests in `CRM_AllTests` extend `CiviUnitTestCase`" or "all the tests in `org.example.foo` extend `PHPUnit_Framework_TestCase`. (This way of thinking probably arose as a defensive reaction all the heaviness in `CiviUnitTestCase`...) Using something else (not `CiviUnitTestsCase`) would make the suite more... nuanced/diverse.

I think your suggestion has better-than-even odds of working, but I don't want to put that issue question in this scope-of-work.